### PR TITLE
OpenDF describe image

### PR DESCRIPTION
**Pull request by ShehanWeerakkody**
- Implemented an image annotation module. It takes an image as the input and categorize it into a particular class based on the content of the image. And it prints the class in the top-left corner. 

- For the classification three pre-trained models were used. All the models were trained using a well defined Convolutional Neural Network. VGG16, VGG19 and ResNet50 are the three models. 

- Following is an output with use of ResNet50 model:
image

### CODE REVIEW

**Comment by ShehanWeerakkody** [comment]
Hi Raveen,
As I mentioned above, I didn't limited to a specific pre-trained model. I just selected three models and implement the image annotation module. As an example, I attached the output image of ResNet50 model. The best model will be selected based on accuracy and speed.
